code/math/staticrand.cpp

```diff
@@ -7,26 +7,21 @@
  *
 */ 
 
-
-
 #include "math/staticrand.h"
 #include "math/vecmat.h"
 
-
-int Semirand_inited = 0;
+bool Semirand_inited = false;
 int Semirand[SEMIRAND_MAX];
```

Comment:
If this is only used internally, in code sections that now constantly convert it to unsigned, why not just make it an array of unsigned ints?

Reply:
true,
in my first try, I converted the whole interface to unsigned.
When I reverted that, I missed this bit.